config/config.exs

```diff
@@ -112,6 +112,12 @@ config :ueberauth, Ueberauth,
     cognito: nil
   ]
 
+# Sentry for error tracking
+config :sentry,
+  dsn: {:system, "SENTRY_DSN"},
+  included_environments: [:prod],
+  environment_name: Mix.env()
```

Comment:
Would putting `config :sentry, dsn: {:system, "SENTRY_DSN"}` only in `prod.exs` accomplish the same thing?

Reply:
It would. The Sentry docs present several ways of getting to this same end, and that's another way. I did it like this because it was convenient for local testing: I was able to add `:dev` as an `included_environment` and generate errors locally (see e.g. https://sentry.io/organizations/mbtace/issues/1755323107/?project=5303878&query=&statsPeriod=14d).

If you have a strong preference for putting this in `prod.exs`, that's fine by me. I would also be fine leaving this as-is.